cnv/cnv_virtual_machines/cnv-create-vms.adoc

```diff
@@ -10,7 +10,7 @@ Use one of these procedures to create a virtual machine:
 * Running the virtual machine wizard
 * Pasting a pre-configured YAML file with the virtual machine wizard
 * Using the CLI
-* Importing a VMware virtual machine or template with the virtual machine wizard
+* xref:../../cnv/cnv_virtual_machines/cnv_importing_vms/cnv-importing-vmware-vm.adoc#cnv-importing-vmware-vm[Importing a VMware virtual machine or template with the virtual machine wizard]
```

Comment:
Kind of a semantic nit:
- Is the template import the relevant piece here, because a new VM gets created from the template? If so, I'd change the link text so that it just refers to importing templates.